protocol/src/protocol.ts

```diff
@@ -1207,6 +1207,26 @@ export namespace ShowMessageRequest {
 	export const type = new RequestType<ShowMessageRequestParams, MessageActionItem | null, void, void>('window/showMessageRequest');
 }
 
+export interface ShowTextDocumentRequestParams {
+	/**
+	 * A name of a file on disk.
+	 */
+	fileName: string;
```

Comment:
This should be 

```
textDocument: TextDocumentIdentifier;
```

Reply:
Agreed. This needs to be a URI and not the name of a file.

Reply:
Fixed, thanks.